SourcePointMetaApp/SourcePointMetaAppTests/ConsentViewControllerTests/ConsentTest.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ConsentTest.swift
+//  ConsentViewControllerTests
+//
+//  Created by Vilas on 9/11/19.
+//
+
+import XCTest
+@testable import ConsentViewController
+
+class ConsentTest: XCTestCase {
+    
+     var consent : Consent?
+
+    override func setUp() {
+        consent = Consent(id: "5b76ebcd5582409726cdd85", name: "Facebook")
+    }
+
+    override func tearDown() {
+    }
+    
+    /// this test method is used to test consent string method
+    func testConsentDescription() {
+        if !(consent?.description.isEmpty)! {
```

Comment:
I think here we could assert that `consent.description == "Consent(id: 5b76ebcd5582409726cdd85, name: Facebook"`.

Reply:
Done, If we change the format of description in future then we have to change test case also thats why I was just checking whether the description is empty or not. by this way also we can conclude that we are getting consent data or not.